nixos/tests/nitter.nix

```
@@ -0,0 +1,19 @@
+import ./make-test-python.nix ({ pkgs, ... }:
+
+{
+  name = "nitter";
+  meta.maintainers = with pkgs.lib.maintainers; [ erdnaxe ];
+
+  nodes.machine = {
+    services.nitter = {
+      enable = true;
+      config.hmacKey = "unsecured";
+    };
+  };
+
+  testScript = ''
+    machine.wait_for_unit("nitter.service")
+    machine.wait_for_open_port("8080")
+    machine.succeed("curl http://localhost:8080/")
```

```suggestion
    machine.succeed("curl --fail http://localhost:8080/")
```
otherwise curl succeeds on 404...